deploy/scripts/src/authentication/test.ts

```diff
@@ -203,14 +211,20 @@ export function signUp(): void {
   const mfaOption: mfaType = Math.random() <= 0.5 ? 'SMS' : 'AUTH_APP'
   iterationsStarted.add(1)
 
-  // B01_SignUp_01_OrchStubSubmit
-  res = orchStubSubmit(groups)
+  // B01_SignUp_01_StubSubmit based on the route i.e. RP or ORCH
+  if (route === 'ORCH') {
+    res = orchStubSubmit(groups)
+  } else if (route === 'RP') {
+    res = rpStubSubmit(groups)
+  } else {
+    throw new Error(`Invalid route: ${route}`) // Handle invalid route values
+  }
```

Comment:
Could use a ternary operator instead of the if-else block.

```suggestion
  route == 'ORCH' ? (res = orchStubSubmit(groups)) : (res = rpStubSubmit(groups))
```

Reply:
added
res = route == 'ORCH' ? (res = orchStubSubmit(groups)) : (res = rpStubSubmit(groups))

Reply:
I'm getting an error related to the line 
'route == 'ORCH' ? (res = orchStubSubmit(groups)) : (res = rpStubSubmit(groups))'. 
The error message is:
"Expected an assignment or function call and instead saw an expression.eslint[@typescript-eslint/no-unused-expressions](https://typescript-eslint.io/rules/no-unused-expressions)"